jsdrupal/admin_ui
# [#9] Adds a JavaScript application and a router that will render it

This PR adds a simple application built with Parcel as an example for now.

Testing instructions:
- Enable this module
- Run `yarn build` from the module root
- Visit `/hello-world` and see the example component rendered
